esm/hyper/wire.js

```diff
@@ -10,13 +10,13 @@ import render from './render.js';
 // all wires used per each context
 const wires = new WeakMap;
 
-// A wire is a callback used as tag function
+// A wire is a callback used as tag a function
```

Comment:
the `tag function` is a concept itself described also in MDN as such

Reply:
I agree with you, but I'm trying to figure out how to remove that "a" so it is back to the way it was.

Reply:
I couldn't figure out how to edit this in the review stage to take out the "a". Instead I submitted another edit of just the main comment I was trying to enhance. Please have patients with me. I will be more helpful once I learned the fundamentals of github.